### ISSUE TITLE
PolymerElement

### ISSUE THREAD

**Issue by lilletech** [vaadin/gwt-polymer-elements#44]
Sometimes, when your init the widget via uibinder and call the setValue() of a input does'nt work. the console shows no errors but the value stays empty.

Calling setValue() via a Function passed in the method ready()  fixes that.
Is there a method to get notified when all the polymer elements of a container are ready ?

**Comment by manolo** [comment]
Do you use Elements or Widgets in your UiBinder ?

**Comment by lilletech** [comment]
I'm using widgets in uibinder

**Comment by manolo** [comment]
The ways to get notified when the ui is ready are:
- To know that all the imports are loaded you can use:

```
   Polymer.whenReady(Function callback);
```
- To know when a specific widget or element is ready use:

```
   Polymer.ready(Element el, Function callback);
```

**Comment by Mattallica** [comment]
Note that the whenReady method is available since the 1.2.1.0 version of vaadin-gwt-polymer-elements
